README.md

```diff
@@ -20,7 +20,7 @@ pre-commit install
 export WEB3_INFURA_PROJECT_ID=<Infura project ID>
 export APE_ACCOUNTS_BOT_PASSPHRASE=<Passphrase for account with alias BOT>
```

Comment:
```suggestion
export APE_ACCOUNTS_BOT_PASSPHRASE=<Passphrase for account with alias BOT>
export ETHERSCAN_API_KEY=<API Key for Etherscan>
```

Reply:
if you don't mind to have it all in one PR, otherwise I'll put these changes in separate one

Reply:
No problem at all! Does it look good now?